src/Tests/Microsoft.NET.Publish.Tests/GivenThatAPublishedDepsJsonShouldContainVersionInformation.cs

```diff
@@ -255,8 +255,10 @@ string GetRollForwardNetCoreAppVersion()
                 IsSdkProject = true,
                 IsExe = true
             };
-            var testAsset = _testAssetsManager.CreateTestProject(testProject)
-                .Restore(Log, testProject.Name);
+            var testAsset = _testAssetsManager.CreateTestProject(testProject);
+            NuGetConfigWriter.Write(testAsset.TestRoot, NuGetConfigWriter.DotnetCoreMyGetFeed);
```

Comment:
@johnbeisner Without adding this feed I get an error on CI: `Unable to find package Microsoft.NETCore.App with version (>= 2.1.0-preview3-26411-06`

Should this package not be in the dotnet-core blob storage feed?